installer/def.bzl

```diff
@@ -91,14 +103,16 @@ def installer(name, data, executable = True, target_subdir = ""):
 
     Args:
       name: A unique name of this rule.
+      compilation_mode: If not empty, sets compilation_mode of targets in data.
```

Comment:
Apologies for not getting to this sooner! Just want to note that you're not only setting compilation_mode of targets in data but also of that entire subtree that is rooted in these data deps.

Reply:
Thanks for taking a look! Yes, this is intended - I want to make it easy to do the right thing and install an optimized build.